### PR TITLE
Add sessions support

### PR DESCRIPTION
**Pull request by pradel**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -134,6 +138,41 @@ class Mongo {
       throw new Error('User not found');
     }
   }
+
+  async createSession(userId: string, ip: string, userAgent: string): Promise<SessionType> {
+    const ret = await this.sessionCollection.insertOne({
+      userId,
```
Comment (pradel) [inline src/index.js]: Should we convert `userId` to `objectId`?

**Comment by TimMikeladze** [comment]
Please update the package versions for accounts server and common in to 0.0.2 in package json.

**Comment by pradel** [comment]
Done

File: src/index.js (hunk 1 above)
Comment (TimMikeladze) [inline src/index.js]: Uh I'm not sure. I don't think it matters since user id is  already guaranteed to be unique.

Comment (pradel) [inline src/index.js]: "It is good to note that between the ObjectId (http://docs.mongodb.org/manual/reference/object-id/) and it's hex representation there is in fact 12 bytes of difference, the ObjectId being 12 bytes and it's hex representation being 24."
http://stackoverflow.com/questions/27896979/difference-between-storing-an-objectid-and-its-string-form-in-mongodb

It take less size in db and allow to have better performance on indexes.

**Comment by TimMikeladze** [comment]
Looks good.

**Pull request merged**